pyglmnet/pyglmnet_utils.py

```diff
@@ -0,0 +1,149 @@
+from copy import deepcopy
+import numpy as np
+from scipy.special import expit
+
+def softmax_(w):
```

Comment:
what's the convention for hidden functions/methods? i would want to make these private later (related to issue #50)